ProjectSettings/ProjectSettings.asset

```diff
@@ -1035,4 +1051,4 @@ PlayerSettings:
   hmiLoadingImage: {fileID: 0}
   platformRequiresReadableAssets: 0
   virtualTexturingSupportEnabled: 0
-  insecureHttpOption: 0
+  insecureHttpOption: 2
```

Comment:
Hm. Please cross check - as some of our servers do not have ssl active? Is this the option that allows or forbids insecure http?

Reply:
![image](https://github.com/user-attachments/assets/5f60f19e-1a47-4655-ab2d-c76094ec424d)